src/os/bluestore/BlueStore.cc

```diff
@@ -9160,6 +9160,142 @@ int BlueStore::expand_devices(ostream& out)
   return r;
 }
 
+int BlueStore::expand_devices_online(ostream& out)
```

Comment:
What's about making one more step forward and get rid off the legacy expand_devices() and use the new one from ceph_bluestore_tool as well? Obviouly after proper device mount..